examples/taskruns/cloud-event.yaml

```diff
@@ -75,6 +74,30 @@ spec:
       - name: notification
         type: cloudEvent
   steps:
+    - name: wait-for-sink
+      image: python:3-alpine
+      imagePullPolicy: IfNotPresent
+      command: ["/bin/sh"]
+      args:
```

Comment:
Do we want to use `script:` here? Or maybe wait until it's actually released in 0.9?

Reply:
I can prepare the PR, we can decide to merge it later, but it would be good to have updated examples when 0.9 goes out.